spec/views/users/index.html.erb_spec.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'users/index.html.erb', type: :view do
+  let(:users) do
+    [
+      FactoryBot.create(:user),
+      FactoryBot.create(:user, email: 'Joe@joe.de')
```

Comment:
does these two do the same? except that the second user has a different email?

Reply:
Exactly

Reply:
You might consider using `Faker::Internet.safe_email`
https://github.com/stympy/faker/wiki/Internet